src/main/java/seedu/contax/logic/commands/RangeCommand.java

```diff
@@ -58,22 +59,30 @@ public RangeCommand(Index fromIndex, Index toIndex, String commandInput) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-
         if (fromIndex.getZeroBased() > toIndex.getZeroBased()) {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
         List<CommandResult> commandResultList = new ArrayList<>();
+        Person restorePerson = model.getFilteredPersonList().get(toIndex.getZeroBased());
         for (int i = toIndex.getOneBased(); i >= fromIndex.getOneBased(); i--) {
             AddressBookParser addressBookParser = new AddressBookParser();
             try {
                 String commandText = ParserUtil.parseAndCreateNewCommand(commandInput, Integer.toString(i));
-
                 logger.info("----------------[RANGE COMMAND][" + commandText + "]");
-
                 Command command = addressBookParser.parseCommand(commandText);
-                commandResultList.add(command.execute(model));
+                try {
+                    commandResultList.add(command.execute(model));
+                } catch (CommandException ce) {
+                    commandResultList.clear(); // only for the purpose of not changing output in feature freeze
+                    // special case: duplicated name editing, restore to previous name
+                    if (fromIndex.getZeroBased() != toIndex.getZeroBased() && commandText.startsWith("editperson")) {
```

Comment:
Similar to the above comment, try not to hardcode `editperson`.

Reply:
thx changed